packages/desktop-client/src/components/reports/graphs/showActivity.ts

```diff
@@ -33,22 +37,32 @@ export function showActivity({
   endDate,
   field,
   id,
+  interval = 'Day',
 }: showActivityProps) {
   const amount =
-    balanceTypeOp === 'totalDebts' || type === 'debts' ? 'lte' : 'gte';
+    balanceTypeOp === 'totalDebts' || type === 'debts' ? true : false;
   const hiddenCategories = categories.list.filter(f => f.hidden).map(e => e.id);
   const offBudgetAccounts = accounts.filter(f => f.offbudget).map(e => e.id);
+  const fromDate =
+    interval === 'Weekly'
+      ? 'dayFromDate'
+      : (((ReportOptions.intervalMap.get(interval) || 'Day').toLowerCase() +
+          'FromDate') as 'dayFromDate' | 'monthFromDate' | 'yearFromDate');
+  let isDateOp = false;
+  if (interval === 'Weekly' || type !== 'time') {
+    isDateOp = true;
+  }
```

Comment:
Can probably simplify this down to a single assignment.  And then make it const.
```suggestion
  const isDateOp = interval === 'Weekly' || type !== 'time';
```